packages/codegen-ui-golden-files/lib/react/forms/datastore-create-form-with-belongs-to/MemberCreateForm.jsx

```diff
@@ -0,0 +1,359 @@
+                  }}
+                />
+              </Badge>
+            );
+          })}
+        </ScrollView>
+      )}
+      <Divider orientation="horizontal" marginTop={5} />
+    </React.Fragment>
+  );
+}
+export default function MyMemberForm(props) {
+  const {
+    clearOnSuccess = true,
+    onSuccess,
+    onError,
+    onSubmit,
+    onCancel,
+    onValidate,
+    onChange,
+    overrides,
+    ...rest
+  } = props;
+  const initialValues = {
+    name: undefined,
+    team: undefined,
+  };
+  const [name, setName] = React.useState(initialValues.name);
+  const [team, setTeam] = React.useState(initialValues.team);
+  const [errors, setErrors] = React.useState({});
+  const resetStateValues = () => {
+    setName(initialValues.name);
+    setTeam(initialValues.team);
+    setCurrentTeamValue(undefined);
+    setCurrentTeamDisplayValue(undefined);
+    setErrors({});
+  };
+  const [currentTeamDisplayValue, setCurrentTeamDisplayValue] = React.useState(undefined);
+  const [currentTeamValue, setCurrentTeamValue] = React.useState(undefined);
+  const teamRef = React.createRef();
+  const teamRecords = useDataStoreBinding({
+    type: 'collection',
+    model: Team,
+  }).items;
+  const getDisplayValue = {
+    team: (record) => record?.id,
+  };
+  const validations = {
+    name: [],
+    team: [],
+  };
+  const runValidationTasks = async (fieldName, currentValue, getDisplayValue) => {
+    const value = getDisplayValue ? getDisplayValue(currentValue) : currentValue;
+    let validationResponse = validateField(value, validations[fieldName]);
+    const customValidator = fetchByPath(onValidate, fieldName);
+    if (customValidator) {
+      validationResponse = await customValidator(value, validationResponse);
+    }
+    setErrors((errors) => ({ ...errors, [fieldName]: validationResponse }));
+    return validationResponse;
+  };
+  return (
+    <Grid
+      as="form"
+      rowGap="15px"
+      columnGap="15px"
+      padding="20px"
+      onSubmit={async (event) => {
+        event.preventDefault();
+        let modelFields = {
+          name,
+          team,
+        };
+        const validationResponses = await Promise.all(
+          Object.keys(validations).reduce((promises, fieldName) => {
+            if (Array.isArray(modelFields[fieldName])) {
+              promises.push(
+                ...modelFields[fieldName].map((item) =>
+                  runValidationTasks(fieldName, item, getDisplayValue[fieldName]),
+                ),
+              );
+              return promises;
+            }
+            promises.push(runValidationTasks(fieldName, modelFields[fieldName], getDisplayValue[fieldName]));
+            return promises;
+          }, []),
+        );
+        if (validationResponses.some((r) => r.hasError)) {
+          return;
+        }
+        if (onSubmit) {
+          modelFields = onSubmit(modelFields);
+        }
+        try {
+          await DataStore.save(new Member(modelFields));
```

Comment:
I could be wrong but would this be necessary create a bi-directional relationship without doing anything to the team? Like,
```
await DataStore.save(Team.copyOf(team, team => team.Members.push[newMember]));
```